Sprint-3/revise/implement/count.js

```diff
@@ -0,0 +1,22 @@
+function countChar(str, char) {
+  let count = 0;
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] === char) {
+      count++;
+    }
+  }
+  return count;
+}
+
+// Test cases
+
+console.log(countChar("aaaaa", "a"));
```

Comment:
If you are interested in preventing some code (like those custom testing code) in a file containing the function definition get executed when the file is imported by another script using `require(...)`, you can try asking ChatGPT "how to check if a JS script is executed directly or via import?".